test/unit/test_counter.dart

```diff
@@ -28,18 +29,20 @@ abstract class SubCounterActions extends ReduxActions {
 }
 
 void _increment(Counter state, Action<int> action, CounterBuilder builder) =>
-    builder.count = state.count + action.payload;
+    builder.count += action.payload;
```

Comment:
Note that this change to the test is needed because the second time, the state will not be the same as the builder, as the state is not updated between dispatching reducers. That's just a problem because a piece of state is updated multiple times.